lasagne/utils.py

```diff
@@ -25,10 +48,21 @@ def shared_empty(dim=2, dtype=None):
 
 
 def as_theano_expression(input):
-    """
-    Wraps the given input as a Theano constant if it is not
+    """Wrap as Theano expression.
+
+    Wraps the given input as a Theano Constant if it is not
```

Comment:
Ah, yes, I didn't want to confuse it -- a TensorConstant -- with a "normal" constant. But it's probably better without any capitalisation. I'll remove it.